crates/sui-framework/tests/governance_test_utils.move

```diff
@@ -59,6 +60,7 @@ module sui::governance_test_utils {
         validators: vector<Validator>, sui_supply_amount: u64, storage_fund_amount: u64, ctx: &mut TxContext
     ) {
         sui_system::create(
+            object::new(ctx), // it doesn't matter what ID sui system state has in tests
```

Comment:
Are we sure this is the case?

Reply:
If we add the assert in `sui_system::create` then it's not the case. but in other places the ID of this isn't checked,